package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "wagtail",
+  "version": "1.0.0",
```

Comment:
Presumably we'll want to keep this up to date with the main wagtail version number, so will want to document release steps to update it. However I suppose given that wagtail won't be distributed as a node package, this is slightly less vital.